tests/suites/test_suite_pk.function

```diff
@@ -423,6 +423,9 @@ void pk_rsa_overflow( )
     mbedtls_pk_context pk;
     size_t hash_len = (size_t)-1;
 
+    if( sizeof( size_t ) <= sizeof( unsigned int ) )
```

Comment:
Technically this test is wrong, it should be `SIZE_T_MAX < UINT_MAX`, since we care about the value range and not the storage size. In theory `size_t` or `unsigned int` could have padding bits, even though in practice they don't.

The test could also be `(size_t) -1 < (unsigned int) -1` to keep the type names inside the code for clarity.